src/orchard/cljs/analysis.cljc

```diff
@@ -0,0 +1,206 @@
+(ns ^{:doc "ClojureScript analysis functions."
+      :author "Gary Trakhman"
+      :added "0.6.0"}
+ orchard.cljs.analysis
+  (:require [orchard.misc :as u]
+            #?(:clj [cljs.repl]))
+  (:refer-clojure :exclude [find-ns find-var all-ns ns-aliases]))
+
+(def NSES :cljs.analyzer/namespaces)
```

Comment:
I guess that's meant to be private, right? In general I don't see much point in mapping a constant to a keyword.

Reply:
Yes, you are right, it was like that but I can change it